Makefile

```diff
@@ -1211,7 +1211,7 @@ filter_bench: $(OBJ_DIR)/util/filter_bench.o $(LIBRARY)
 	$(AM_LINK)
 
 ifeq ($(DEBUG_LEVEL),0)
-db_stress: $(OBJ_DIR)/db_stress_tool/db_stress.o $(STRESS_OBJECTS) $(LIBRARY)
+db_stress: $(OBJ_DIR)/db_stress_tool/db_stress.o $(STRESS_OBJECTS) $(TESTUTIL) $(LIBRARY)
```

Comment:
Can you explain why this change?

Reply:
Fix release build. I think it's more implicitly understandable than much of the Makefile. If you remove it and "cover" the line in your build, you're guaranteed a linker error. And

db_stress_tool/db_stress_common.h:72:#include "test_util/testutil.h"

A TODO item from this is to cover the release build in CI

Reply:
> A TODO item from this is to cover the release build in CI

Actually, we have it. https://app.circleci.com/pipelines/github/facebook/rocksdb/2037/workflows/7da6367b-5f7b-4ce9-9c15-8aa65c746266/jobs/4511  I have no idea why it passed in CircleCI because it fails for me every way I try "make release" on that revision